homeassistant/components/august/__init__.py

```diff
@@ -300,13 +326,15 @@ def _update_locks(self):
         self._update_locks_status()
         self._update_locks_detail()
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
+    @Throttle(MIN_TIME_BETWEEN_LOCK_STATUS_UPDATES)
     def _update_locks_status(self):
         status_by_id = {}
         state_by_id = {}
+        last_status_update_by_id = {}
 
         _LOGGER.debug("Start retrieving lock and door status")
         for lock in self._locks:
+            update_start_time = datetime.now()
```

Comment:
If this time will only be used internally in this integration can we use utc now instead?

There are helpers for `now` and `utcnow` in util/dt.py. We prefer to use those.

Reply:
@MartinHjelmare I'll do some testing to see if it works.  We have to compare this with an object that comes back from py-august and there may be an issue with timezoned vs non-timezoned dates

Reply:
Changing to utcnow results in TypeError: can't compare offset-naive and offset-aware datetimes.  I'll see if I can get it sorted out